platform_versioned_docs/version-23.3/enterprise_templates/k8s/groundswell.yml

```diff
@@ -0,0 +1,137 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: tower-groundswell-cfg
+data:
+  # Server settings
+  SWELL_SERVER_HOST: "0.0.0.0"
+  SWELL_SERVER_PORT: "8090"
+
+  # API settings
+  SWELL_API_TRAIN_TIMEOUT: "60"
+  SWELL_API_TRAIN_BATCH_SIZE: "1000"
+  SWELL_API_PREDICT_FRACTIONAL_CPUS: "false"
+
+  # Database settings, different from the Tower DB credentials.
+  # If using Amazon RDS or similar managed database services, `SWELL_DB_URL` will have the form 
+  # SWELL_DB_URL=mysql://db1.abcdefghijkl.us-east-1.rds.amazonaws.com:3306/swell or similar
+  SWELL_DB_URL: mysql://<ANOTHER_DB_AND_PORT>/swell
+  SWELL_DB_USER: swell
+  SWELL_DB_PASSWORD: <YOUR GROUNDSWELL DATABASE PASSWORD>
+  SWELL_DB_DIALECT: mysql
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: groundswell
+  labels:
+    app: groundswell
+spec:
+  selector:
+    app: groundswell
+  ports:
+    - name: http
+      port: 8090
+      # targetPort must match with the SWELL_SERVER_PORT in the ConfigMap above.
+      targetPort: 8090
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: groundswell
+  labels:
+    app: groundswell
+spec:
+  selector:
+    matchLabels:
+      app: groundswell
+  strategy:
+    type: Recreate
+
+  template:
+    metadata:
+      labels:
+        app: groundswell
+    spec:
+      imagePullSecrets:
+        - name: "cr.seqera.io"
```

Comment:
I didn't check, but are there instructions in the previous steps on how to create this image pull secret?

Reply:
The instructions are provided here: https://docs.seqera.io/platform/24.1/enterprise/kubernetes#pipeline-optimization.